{"name": "test2_login_with_error_password", "status": "passed", "attachments": [{"name": "stdout", "source": "9517a89a-6dcc-4327-a693-a0bdb4c6b95b-attachment.txt", "type": "text/plain"}], "start": 1690963736100, "stop": 1690963751536, "uuid": "956b3c1a-062d-4cd0-944e-9e3e5e3d2454", "historyId": "a59fe43aa9d3514c6555f810b0694901", "testCaseId": "a59fe43aa9d3514c6555f810b0694901", "fullName": "testcase.login_logout.test_login_with_error_info.TestErrorLogin#test2_login_with_error_password", "labels": [{"name": "severity", "value": "minor"}, {"name": "story", "value": "login with error password"}, {"name": "feature", "value": "Login error"}, {"name": "parentSuite", "value": "testcase.login_logout"}, {"name": "suite", "value": "test_login_with_error_info"}, {"name": "subSuite", "value": "TestErrorLogin"}, {"name": "host", "value": "Joys-MacBook-Pro.local"}, {"name": "thread", "value": "28553-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.login_logout.test_login_with_error_info"}]}